modules/ROOT/pages/_partials/commons/common-database.adoc

```diff
@@ -45,11 +45,11 @@ The most basic structure is to use the single default scope with a single defaul
 .Couchbase Lite Examples
 include::partial$diagram-definitions.adoc[tag=couchbase-lite-example-structure]
 
-If a scope is not explicitly referenced in calls to the database, then scope is assumed to `_default`. Likewise, if no collection name is given then then the `_default` collection is assumed.
+If a scope is not explicitly referenced in calls to the database, then scope is assumed to `_default`. Likewise, if no collection name is given then the `_default` collection is assumed.
```

Comment:
Might as well quickfix to ventilate prose (e.g. put new sentence "Likewise..." on a new line)